.travis.yml

```diff
@@ -35,10 +35,12 @@ rust:
 
 before_script:
   - pip install 'travis-cargo<0.2' --user && export PATH=$HOME/.local/bin:$PATH
+  - rustup component add rustfmt-preview
   - cargo install cargo-travis || echo "Skipping cargo-travis installation as it is already cached"
   - export PATH=$HOME/.cargo/bin:$PATH
 
 script:
+  - cargo fmt --all -- --check
```

Comment:
Would it better to move format check after running all of the tests?

Reply:
Yes, we could - though it won't make much difference since if test fails the format check won't run.

Reply:
Fair enough. One could always run tests or format locally. I was thinking if it would be a good idea to add a paragraph with a command to run format. Either way - LGTM.